design/src/production-chains/raw-resources.md

```diff
@@ -0,0 +1,201 @@
+# Raw Resources
+
+These are the simple, fundamental materials that intermediate and end products are made of.
+Raw resources cannot be derived from another raw resources: they are "elemental" in that way.
+The total amount of each raw resource (other than water and energy) is conserved at each resource processing step.
+
+Different areas of the map will have different quantities of each resource to begin with.
+
+## Core
+
+Resources that should be added in an MVP demo.
+
+### Light
+
+**Acquisition:** It comes from the sun.
+
+**Uses:** Used by plants to capture energy, and provide calories to entire colony.
+
+**Handling properties:** Cannot be tranported. Blocked by shade.
+
+**Storage properties:** Cannot be stored.
+
+**Path to renewability:** The sun really does just keep burning.
+
+**Downsides of excess:** Heat, sun scorch of plants.
+
+### Water
+
+**Acquisition:** Gather from lakes, rivers, soil and rain.
+
+**Uses:** Essential for sustenance of all life. One of the primary transport mechanics of goods.
+
+**Handling properties:** Sinks into soil. Flows downstream. Cannot be carried directly, as it is a fluid.
+
+**Storage properties:** Evaporates over time.
+
+**Path to renewability:** Rainfall, incoming streams.
+
+**Downsides of excess:** Washes away goods. Drowns structures over time.
+
+### Dirt
+
+Planned to split into sand/silt/clay/organic matter after the MVP.
+
+**Acquisition:** Dig up the ground.
+
+**Uses:** Build simple structures. Grow plants and fungi.
+
+**Handling properties:** Solid object.
+
+**Storage properties:** Solid object.
+
+**Path to renewability:** Filter incoming streams.
+
+**Downsides of excess:** Piles up cluttering movement.
+
+### Stone
+
+May split into igneous/sedimentary/metamorphic rock if enough distinct uses arise.
+
+**Acquisition:** Dig up the ground.
+
+**Uses:** Build more durable structures.
+
+**Handling properties:** Solid object.
+
+**Storage properties:** Solid object.
+
+**Path to renewability:** Mines.
+
+**Downsides of excess:** Piles up cluttering movement.
+
+### Nutrients
+
+Planned to split into at least nitrogen and phosphorus after the MVP.
+
+**Acquisition:** Found in soil. Can be recycled.
+
+**Uses:** Essential for growing organisms.
+
+**Handling properties:** Solid object.
+
+**Storage properties:** If left on ground, will be absorbed by soil.
+
+**Path to renewability:** Recycling.
+
+**Downsides of excess:** Nutrient toxicity stunts plant growth.
+
+## Planned
+
+### Nutrients - Nitrogen
+
+**Acquisition:** Found in soil, nitrogen fixing organisms.
+
+**Uses:** Essential for vegetative growth.
+
+**Handling properties:** Solid, but water soluble.
+
+**Storage properties:** Water will wash away. Fire / explosion risk?
```

Comment:
I wonder if it makes sense to have nutrients such as nitrogen/phosphorus stored in their pure form

Usually, they are stored in living organisms as part of other organic molecules, right? Pure/refined storage is mostly a human thing?

Also adds an interesting mechanic: in storage, you will typically not store refined goods, but storage-conducive goods, which you later have to filter/react for actual use

Reply:
Yeah, my expectation is that you'll probably end up storing balls of fertilizer in some form.

> Also adds an interesting mechanic: in storage, you will typically not store refined goods, but storage-conducive goods, which you later have to filter/react for actual use

Yes!